src/odemis/util/test/spot_test.py

```diff
@@ -333,7 +333,8 @@ def test_multiple(self):
 
         # Map the found spot locations to the known spot locations.
         tree = scipy.spatial.cKDTree(loc)
-        distances, indices = tree.query(ji, k=1, workers=-1)
+        # NOTE: Starting SciPy v1.6.0 the `n_jobs` argument will be renamed `workers`
```

Comment:
From the doc: 
>Changed in version 1.6.0: The “n_jobs” argument was renamed “workers”. The old name “n_jobs” is deprecated and will stop working in SciPy 1.8.0.

So we have a tiny bit of margin as the current version of scipy is 1.7.